ui/StatusQ/src/keychain_osx.mm

```diff
@@ -0,0 +1,239 @@
+#include "StatusQ/keychain.h"
+
+#include <QDebug>
+#include <QEventLoop>
+#include <QFuture>
+#include <QtConcurrent/QtConcurrent>
+
+#include <Foundation/Foundation.h>
+#include <LocalAuthentication/LocalAuthentication.h>
+#include <Security/Security.h>
+
+const static auto authPolicy = LAPolicyDeviceOwnerAuthentication;
```

Comment:
We should probably use this instead:
```cpp
 const static auto authPolicy = 
 #if defined(MAC_OS_VERSION_15_0) && MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_VERSION_15_0 
     LAPolicyDeviceOwnerAuthenticationWithBiometricsOrCompanion; 
 #elif defined(MAC_OS_X_VERSION_10_15) && MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_15 
     LAPolicyDeviceOwnerAuthenticationWithBiometricsOrWatch; 
 #else 
     LAPolicyDeviceOwnerAuthenticationWithBiometrics 
 #endif 
```

But for now I'm keeping the current solution, which allows authenticate with OS password:
https://github.com/status-im/status-desktop/blob/5f4f7470d227ab6574f0d78f10a45849f7c8e520/vendor/DOtherSide/lib/src/Status/LocalAuthentication.mm#L132